.editorconfig

```diff
@@ -0,0 +1,48 @@
+# https://EditorConfig.org
+
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+indent_style = space
+indent_size = 4
+
+# Ignore paths
+[/tests/data-files/**]
+charset = unset
+end_of_line = unset
+insert_final_newline = unset
+trim_trailing_whitespace = unset
+indent_style = unset
+indent_size = unset
```

Comment:
added to ignore files like `tests/data-files/S1A_20200103_17RMJ_ASC/Gamma0_VH.vrt` etc